docs/docs/modeling.md

```diff
@@ -108,6 +107,25 @@ classDiagram
 
 ## Category - カテゴリー
```

Comment:
Should we make the maximum number of `Task` that a `Category` can have?

Reply:
Thank you for your advice. We need to talk about that.